.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
+package-lock.json
```

Comment:
It is actually good to have the latest package-lock committed/pushed on to the repo. There are cases when someone else (person/heroku) is generating it by running 'npm i' and might get some sub-dependencies in other versions installed which may cause service to not work correctly. If you want I can explain it in details, just hit me up.